services/currency-notifier/internal/context/app_context.go

```diff
@@ -54,7 +54,7 @@ func (ctx *AppContext) Init() {
 		util.GetEnv("RABBITMQ_URL", "amqp://username:password@localhost:5672/"),
 	)
 	ctx.MessagePublisher.Connect()
-	log.Printf("Connected to RabbitMQ")
+	zap.S().Info("Connected to RabbitMQ")
```

Comment:
why u use it as signleton, better create logger instance and use it as dependency all over the project? 
how u will test it?

think abt interfaces